### PR TITLE
Pr fix nfs roles

### PR DESCRIPTION
**Pull request by lerwys**
Along with general changes, this mainly fixes the nfsserver to export the current directories. There is still the change of updating the NFS server directories to come.

### CODE REVIEW

File: group_vars/all [hunk 1]
```diff
@@ -3,9 +3,14 @@
 nfs_server_control_room: 10.128.255.9
 nfs_server_control_opi: 10.128.255.9
 
-fac_workstation_ip: 10.0.7.55
+nfs_clients_auto_backup_ta_ti_rack_ip: 10.128.255.5
+nfs_clients_auto_backup_ca_ra_ctrl_ip: 10.128.255.4
+nfs_clients_auto_backup_la_ra_ctrl_ip: "10.128.255.3
```
Comment (anacso17) [inline group_vars/all]: Is `"` a typo?

Comment (lerwys) [reply group_vars/all]: Yes.... Thanks for catching that. I didn't test this role as I didn't want to change anything at the server, but I will test it now on m machine. Thanks again!

**Comment by lerwys** [comment]
@edupcoelho @ito-rafael Can you check if the resulting /etc/exports file is fine? 

The resulting /etc/exports file will look like the following:

```
# /etc/exports: the access control list for filesystems which may be exported
#		to NFS clients.  See exports(5).
#
# Example for NFSv2 and NFSv3:
# /srv/homes       hostname1(rw,sync,no_subtree_check) hostname2(ro,sync,no_subtree_check)
#
# Example for NFSv4:
# /srv/nfs4        gss/krb5i(rw,sync,fsid=0,crossmnt,no_subtree_check)
# /srv/nfs4/homes  gss/krb5i(rw,sync,no_subtree_check)

# --- CONS auto backup ---
/home/con-srv/AutoBackup_TA-TiRack-CO-FWSrv-1 10.128.255.5(rw,sync,no_root_squash)\
                                              *(ro,sync,root_squash)
/home/con-srv/AutoBackup_CA-RaCtrl-CO-Srv-1   10.128.255.4(rw,sync,no_root_squash)\
                                              *(ro,sync,root_squash)
/home/con-srv/AutoBackup_LA-RaCtrl-CO-Srv-1   10.128.255.3(rw,sync,no_root_squash)\
                                              *(ro,sync,root_squash)

# --- CONS OPIs ---
/home/con-srv/cons-opis                       10.128.255.0/24(rw,sync,all_squash,anonuid=1002,anongid=1002)\
                                              10.128.124.0/24(rw,sync,all_squash,anonuid=1002,anongid=1002)\
                                              10.128.254.0/24(rw,sync,all_squash,anonuid=1002,anongid=1002)\
                                              *(ro,sync,root_squash)

# --- lnls-sirius repos ---
/home/nfs-shared/repos-lnls-sirius  10.128.254.0/24(rw,sync,no_subtree_check,no_root_squash,insecure)\
                                    10.0.38.0/24(rw,sync,no_subtree_check,no_root_squash,insecure)

# --- lost and found ---
/home/nfs-shared/lost-and-found     10.128.254.0/24(rw,sync,no_subtree_check,all_squash,insecure,anonuid=1001,anongid=1001)\
                                    10.0.38.0/24(rw,sync,no_subtree_check,all_squash,insecure,anonuid=1001,anongid=1001)

# --- screens-iocs ---
/home/nfs-shared/screens-iocs       10.128.254.0/24(rw,sync,no_subtree_check,all_squash,insecure,anonuid=1001,anongid=1001)\
                                    10.0.38.0/24(rw,sync,no_subtree_check,all_squash,insecure,anonuid=1001,anongid=1001)

# --- backups shared with lnls350-linux ---
/home/nfs-shared/control-system-constants     10.128.254.0/24(rw,sync,no_subtree_check,all_squash,insecure,anonuid=1001,anongid=1001)\
                                              10.0.38.0/24(rw,sync,no_subtree_check,all_squash,insecure,anonuid=1001,anongid=1001)

# --- backups shared with lnls350-linux ---
/home/nfs-shared/backups            10.0.7.55(rw,sync,no_subtree_check,no_root_squash,insecure)

# --- backups shared with lnls350-linux ---
/home/nfs-shared/sirius-epics-screens         10.128.254.0/24(ro,sync,no_subtree_check,all_squash,insecure,anonuid=1001,anongid=1001)\
                                              10.0.38.0/24(ro,sync,no_subtree_check,all_squash,insecure,anonuid=1001,anongid=1001)\
                                              *(ro,sync,no_subtree_check,all_squash,insecure)
```

Would that work for you?

**Comment by ito-rafael** [comment]
@lerwys, @xresende asked us to change the path from where we are mounting the backup dirs, so I'd change the following:

- /home/con-srv/AutoBackup_TA-TiRack-CO-FWSrv-1 
- /home/con-srv/AutoBackup_CA-RaCtrl-CO-Srv-1
- /home/con-srv/AutoBackup_LA-RaCtrl-CO-Srv-1 

To:
- /home/nfs-shared/con-srv/AutoBackup_TA-TiRack-CO-FWSrv-1 
- /home/nfs-shared/con-srv/AutoBackup_CA-RaCtrl-CO-Srv-1
- /home/nfs-shared/con-srv/AutoBackup_LA-RaCtrl-CO-Srv-1 

For us, both paths are fine. Just confirm with @xresende what he wants to do about this.

**Comment by lerwys** [comment]
Hi @ito-rafael . Ok, I will make the changes, but what about `/home/con-srv/cons-opis`? Can I also change it to  `/home/nfs-shared/con-srv/cons-opis`?

**Comment by ito-rafael** [comment]
> Hi @ito-rafael . Ok, I will make the changes, but what about `/home/con-srv/cons-opis`? Can I also change it to `/home/nfs-shared/con-srv/cons-opis`?

@lerwys, you can keep this one for now, please. We will remove this directory soon. When this happens, I'll let you know so you can remove this line and update the file. Ok?

**Pull request merged**